traveler-swift-core/TravelerKit/Models/WishlistToggleError.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  WishlistToggleError.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-09-03.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+public enum WishlistToggleError: Error {
+    case unidentifiedTraveler
+    case unableToAdd
+    case unableToRemove
+    case itemAlreadyInWishlist
+    case notInWishlist
```

Comment:
hmmm

`unableToAdd` and `unableToRemove` ...should these just be a more generic case? What are some circumstances where this can happen?

`itemAlreadyInWishlist` and `notInWishlist` ....does this make sense to even be an error...As a developer would I expect the error callback to get called if I try to wishlist something that is already wishlisted?